### PR TITLE
Just the hint strings

### PR DESCRIPTION
**Pull request by liuche**
Replacing #21

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -404,3 +404,29 @@ msgctxt "exit_firefox_a11y"
...
+#. the name of the app (e.g. Firefox for Fire TV)      When this button is
+#. pressed, the overlay will open and the browser will be hidden underneath it
+#, c-format
+msgctxt "hint_press_menu_to_open_overlay"
+msgid "Press $IMAGE for %1$s Home, search, settings, and more"
+msgstr ""
+
+#, c-format
+msgctxt "hint_press_menu_to_open_overlay_a11y"
+msgid "Press the menu button for %1$s Home, search, settings, and more"
```
Comment (Delphine) [inline locales/templates/app.pot:439]: I'm sorry for the back and forth, but we really need to have a comment explaining what the placeholder stands for in each comment.
Our translation tool shows each string individually with each comment. So for example if there are multiple people localizing, not each one will be working on the same strings and seeing their own comments. In the case someone different was working on this string (vs the one before), the person wouldn't know what the placeholder stands for.
thanks!

**Review by Delphine — CHANGES_REQUESTED** [review]

File: locales/templates/app.pot (hunk 1 above)
Comment (severinrudie) [reply locales/templates/app.pot:439]: Not a problem!  I assume that means the comment explaining the context (e.g., 'Hint text displayed when the browser is open') should also be duplicated?

Comment (Delphine) [reply locales/templates/app.pot:439]: Yes! thank you 😊

**Comment by liuche** [comment]
@Delphine ready for re-review!

**Review by Delphine — APPROVED** [review]
thank you both!

**Pull request merged**